config/default.yml

```diff
@@ -230,7 +230,17 @@ PreCommit:
     install_command: 'npm install -g coffeelint'
     include: '**/*.coffee'
 
-  Credo:
+   CookStyle:
```

Comment:
Indentation looks slightly off here, which may explain why the tests failed.

Reply:
isn't it ironic my updating of a style linter gets caught by a style linting problem